Automattic/newspack-plugin
# Update the Select and Text control with BaseComponent

### All Submissions:






### Changes proposed in this Pull Request:


__Text Control__
* Update margin when multiple controls
* Update disabled style
* remove Muriel colors

__Select Control__
* Reworked/Simplified the component to behave in a similar way as Core, with the Newspack styles

__Wizard Setup: About__
* Fixed the margins between the different form elements

__Before:__

![1 before](https://user-images.githubusercontent.com/177929/70977775-7eaabc80-20a6-11ea-8cf4-9ef0ed89d82a.png)

__After:__

![2 after](https://user-images.githubusercontent.com/177929/70977769-7a7e9f00-20a6-11ea-8c89-5621f48620e0.png)

### How to test the changes in this Pull Request:

1. Check the Components Demo, Setup, and everywhere else you can think of where we have a select. You should see the current style. It looks out of place!
2. Switch to this branch.
3. Recheck the different pages. Do you see the new style?


### Other information:
